src/main/resources/application.yaml

```diff
@@ -72,7 +72,7 @@ robotics:
 
 validation:
   titles: Cllr,Count,DC,Dr,DS,Insp,Lady,Lord,Miss,Mr & Mrs,Mr,Mrs,Ms,PC,Rev,S/M,Sgt
-  dwp_offices: 1,2,3,4,5,6,7,8,9,10,Balham DRT,Birkenhead LM DRT,Chesterfield DRT,Coatbridge Benefit Centre,Inverness DRT,Lowestoft DRT,Milton Keynes DRT,Norwich DRT,Sheffield DRT,Springburn DRT,Watford DRT,Wellingborough DRT,test-hmcts-address
+  dwp_offices: 1,2,3,4,5,6,7,8,9,10,Balham DRT,Birkenhead LM DRT,Chesterfield DRT,Coatbridge Benefit Centre,Inverness DRT,Lowestoft DRT,Milton Keynes DRT,Norwich DRT,Sheffield DRT,Springburn DRT,Watford DRT,Wellingborough DRT,test-hmcts-address,Worthing DRT
```

Comment:
Do we need to add the new dwp office here? https://github.com/hmcts/sscs-evidence-share/blob/master/src/main/resources/dwpAddresses.json

Reply:
@ryandac Not according to the story description!
@jackmaloney could you please clarify?